src/compiler/diagnosticMessages.json

```diff
@@ -2824,5 +2824,9 @@
     "Unknown typing option '{0}'.": {
         "category": "Error",
         "code": 17010
+    },
+    "Too many javascript files in the project. Consider add to the `exclude` list in the config file.": {

```

Comment:
I agree that we should mention both cases, with or without the "exclude" list. Does the following sound better:

"Too many JavaScript files in the project. Use an exact 'files' list, or use the 'exclude' setting  in project configuration to limit included source folders."